### PR TITLE
Fixes #25492 - Remove the :login setting

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

File: app/registries/menu/loader.rb [hunk 1]
```diff
@@ -9,30 +9,26 @@ class Loader
...
           menu.item :locations,          :caption => N_('Locations') if SETTINGS[:locations_enabled]
           menu.item :organizations,      :caption => N_('Organizations') if SETTINGS[:organizations_enabled]
-          if SETTINGS[:login]
-            menu.item :auth_source_ldaps, :caption => N_('LDAP Authentication')
-            menu.item :users,            :caption => N_('Users')
-            menu.item :usergroups,       :caption => N_('User Groups')
-            menu.item :roles,            :caption => N_('Roles')
-          end
+          menu.item :auth_source_ldaps, :caption => N_('LDAP Authentication')
+          menu.item :users,            :caption => N_('Users')
```
Comment (ekohl) [inline app/registries/menu/loader.rb]: Nit: would be nice to keep alignment

Comment (tbrisker) [reply app/registries/menu/loader.rb]: fixed

**Comment by lzap** [comment]
Needs a rebase, refactoring needs to be all green.

**Review by lzap — DISMISSED** [review]

**Comment by tbrisker** [comment]
@lzap fixed the failing tests, all :green_apple: now

**Review by lzap — APPROVED** [review]
Sorry for the delay. Yay! :clap:

**Comment by lzap** [comment]
@tbrisker do you also track installer changes? Probably this needs to be removed from there as well.

**Comment by ekohl** [comment]
Since we always maintain compatibility with older versions (typically n - 2), I'm leaning to not making a change. Foreman will ignore the unused setting and we should remember to remove it when we drop support for 1.20.

**Comment by tbrisker** [comment]
installer side is at https://github.com/theforeman/puppet-foreman/pull/690

**Pull request merged**